### ISSUE TITLE
poor accuracy on NWPU VHR dataset?

### ISSUE THREAD

**Issue by Ismaildata** [ZhAnGToNG1/MSFC-Net#14]
please , i tested your model on NWPU VHR dataset without scaling data and spliting data into 60% test and 40% validation , and i got a poor results using your model mAP 72% , can you tell me what is the problem ?

**Comment by ZhAnGToNG1** [comment]
I think that there are many influencing factors, like 

BatchSize
Learning rate
Data processing: Resize or Crop?
Data splitting
Multi-scale training and testing
